tools/unused.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+
+import os
+import re
+import b
+
+# @run_on_docker
```

Comment:
Don't leave commented code (and this module 100% should not be running on docker)
```suggestion
```